R/RcppExports.R

```diff
@@ -244,6 +244,164 @@ pcnl_rcpp <- function(cell, utility, mum, nests, alpha, mu) {
     .Call(`_m4ma_pcnl_rcpp`, cell, utility, mum, nests, alpha, mu)
 }
 
+#' Two-line Intersection
+#' 
+#' Calculate the intersection between two lines.
+#' 
+#' The first line is defined by \code{P1} and \code{P1}, the second by 
+#' \code{P3} \code{P4}. Returns \code{Inf} when the lines are parallel. If
+#' \code{interior_only} is \code{TRUE}, returns \code{NA} if the intersection
+#' is not within the plane spanning between the two lines.
+#' 
+#' @param P1,P2,P3,P4 Numeric vector with x- and y-coordinates.
+#' @param interior.only,interior_only Logical scalar indicating whether the intersection must
+#' be within the spans of the two lines.
+#' 
+#' @returns A numeric vector with x- and y-coordinates defining the
+#' intersecting point between the two lines.
+#' 
+#' @source Weisstein, Eric W. "Line-Line Intersection.

```

Comment:
once the contribution of 'david sterratt' has been 'redimensioned' this code is good to go

Reply:
I just pushed a commit that addresses this! Let's wait for the checks.

Reply:
FYI: David Sterratt is still listed among the authors of the line.line.intersection

I have approved everything because I don't have much time left today for this
<img width="1509" alt="Screenshot 2022-10-05 at 12 08 27" src="https://user-images.githubusercontent.com/24624390/195792905-9e081d67-9a01-4d20-a3cd-880bc45da3be.png">

Reply:
Ah damn, I think I forgot to regenerate the docs after I removed him. It will disappear it the next PR.